packages/form-js-viewer/src/render/components/form-fields/Select.js

```diff
@@ -38,6 +38,7 @@ export default function Select(props) {
 
   return <div class={ formFieldClasses(type, errors) }>
     <Label
+      id={ prefixId(_id) }
```

Comment:
What exactly is this for?

Reply:
@philippfromme ?

Reply:
It ensures that there will be a `for` attribute on the label referencing the `id` attribute of the form input. I forgot to add it for the `<select>`.

So previously it was

```
<label>Foo</label>
<select id="foo">
  ...
</select>
```

and now it's

```
<label for="foo">Foo</label>
<select id="foo">
  ...
</select>
```